paired.go

```diff
@@ -86,64 +54,80 @@ func (p *paired) String(key string) (*String, error) {
 // Returns a boolean type AMF specified by the key. If the
 // key isn't found it returns a NotFoundError. If it is found
 // but is of the wrong type, this returns a WrongTypeError.
-func (p *paired) Boolean(key string) (*Boolean, error) {
+func (p *Paired) Bool(key string) (*Bool, error) {
 	val, err := p.Get(key)
 	if err != nil {
 		return nil, err
 	}
 
-	if cast, ok := val.(*Boolean); ok {
+	if cast, ok := val.(*Bool); ok {
 		return cast, nil
 	}
 
 	return nil, WrongTypeError
 }
 
-// Returns a number type AMF specified by the key. If the
-// key isn't found it returns a NotFoundError. If it is found
-// but is of the wrong type, this returns a WrongTypeError.
-func (p *paired) Number(key string) (*Number, error) {
-	val, err := p.Get(key)
-	if err != nil {
-		return nil, err
+// Returns an item specified by the key, or returns a NotFoundError.
+func (p *Paired) Get(key string) (AmfType, error) {
+	kb := []byte(key)
+	for _, tuple := range p.tuples {
+		if bytes.Compare(tuple.Key, kb) == 0 {
```

Comment:
[Now](https://github.com/WatchBeam/amf0/pull/2/files#diff-b6cbcc6e2cff4b9fb3ba319ceb66e048L122) you've got it! :stuck_out_tongue_winking_eye: